deploy/lib/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Deploy artifacts from common/deploy/*
+ */
+
+import {
+  ec2Run,
+  ec2Terminate,
+} from './aws-ec2-deploy.js';
+
+// TODO: try to get previous instance id??
+
+// TODO: try to shut down server
+
+// TODO: try to trigger save of current instance runtime data
```

Comment:
- [ ] use volumes and/or snapshots?

Reply:
https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-ec2/Class/AttachVolumeCommand/

https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-ec2/Interface/DetachVolumeRequest/